docs/export/enable-connector.md

```diff
@@ -44,9 +47,9 @@ Use the following configuration as a starting point. Copy and paste it into `exp
 Replace `my_opentsdb_http_instance` with an instance name of your choice, and change the `destination` setting to the IP
 address or hostname of your OpenTSDB database.
 
-Restart your Agent with `service netdata restart` to begin exporting to your OpenTSDB database. Because the
-Agent exports metrics as they're collected, you should start seeing data in your external database after only a few
-seconds.
+Restart your Agent with `service netdata restart` to begin exporting to your OpenTSDB database. The Netdata Agent
```

Comment:
Joel I remember I read some place that nowadays is better to use `systemctl` instead `service` to restart a service on Linuxes (@Ferroin , please, can you confirm this?). If this is truth we should suggest to use `systemctl` or to teach both.

Reply:
Interesting! Yes, let's get confirmation from @Ferroin, but I was under the assumption that `service` is a wrapper that works on systems with both systemd and others, like `init.d`. So, potentially, `service` works on more systems than `systemctl`.

Reply:
You are right. I thought that `service` is deprecated, but
> If there is no suitable legacy action found and COMMAND is one of actions specified in LSB Core Specification, input is redirected to the systemctl.